### PR TITLE
Begin Scaffold of Validator Accounts V2

### PR DESCRIPTION
**Pull request by rauljordan**
**What type of PR is this?**

> Feature

**What does this PR do? Why is it needed?**

As part of our validator accounts refactor, we cannot break backwards compatibility. As such, we will work on `v2` of validator accounts as its own subdirectory of the `validator/accounts` package. This PR sets up that scaffold.

**Which issues(s) does this PR fix?**

Part of #6220

### CODE REVIEW

**Comment by prestonvanloon** [comment]
Is this just a refactoring / renaming of the accounts package? 
What about keymanager? That is going to change significantly.

**Comment by rauljordan** [comment]
Blocked by the accounts design still being in draft mode

**Comment by prestonvanloon** [comment]
Removing ready for review while the status is "blocked"

File: validator/client/polling/propose.go [hunk 1]
```diff
@@ -8,6 +8,7 @@ import (
 	"github.com/pkg/errors"
 	ethpb "github.com/prysmaticlabs/ethereumapis/eth/v1alpha1"
 	"github.com/prysmaticlabs/go-bitfield"
+
```
Comment (rauljordan) [inline validator/client/polling/propose.go:11]:
```suggestion
```

File: validator/keymanager/v1/wallet.go [hunk 2]
```diff
@@ -8,11 +8,12 @@ import (
 	"regexp"
 	"strings"
 
-	"github.com/prysmaticlabs/prysm/shared/bls"
-	"github.com/prysmaticlabs/prysm/shared/bytesutil"
 	e2wallet "github.com/wealdtech/go-eth2-wallet"
 	filesystem "github.com/wealdtech/go-eth2-wallet-store-filesystem"
 	e2wtypes "github.com/wealdtech/go-eth2-wallet-types/v2"
+
```
Comment (rauljordan) [inline validator/keymanager/v1/wallet.go:14]:
```suggestion
```

File: validator/keymanager/v1/remote.go [hunk 3]
```diff
@@ -12,11 +12,12 @@ import (
 
 	"github.com/pkg/errors"
 	ethpb "github.com/prysmaticlabs/ethereumapis/eth/v1alpha1"
-	"github.com/prysmaticlabs/prysm/shared/bls"
-	"github.com/prysmaticlabs/prysm/shared/bytesutil"
 	pb "github.com/wealdtech/eth2-signer-api/pb/v1"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials"
+
```
Comment (rauljordan) [inline validator/keymanager/v1/remote.go:18]:
```suggestion
```

**Pull request merged**